Sources/Moya/MoyaError.swift

```diff
@@ -52,25 +52,36 @@ public extension MoyaError {
 
 extension MoyaError: LocalizedError {
     public var errorDescription: String? {
+        return localizedDescription
```

Comment:
what does localizedDescription return here? Could you add some tests with expected values?

Reply:
With Swift's current implementation of error bridging, this is the same as writing `return errorUserInfo[NSLocalizedDescriptionKey]`. Since `errorDescription` is declared as an optional `String?`, this line could be changed to that to clarify its intent and eliminate possible future issues from Swift implementation changes?

Reply:
adding a comment that it bridges to the NSLocalizedDescriptionKey, and a link to the apple documentation of that would be awesome